docs/examples/track_fusion_example.py

```diff
@@ -0,0 +1,538 @@
+    position_mapping=(0, 2),
+    noise_covar=radar_noise,
+    clutter_model=clutter_model,
+    max_range=3000)  # max_range can be removed and use the default value
+
+# deep copy the first radar specs. Changes in the first one does not influence the second
+radar2 = deepcopy(radar1)
+
+# prepare the first sensor platform and add the sensor
+sensor1_platform = FixedPlatform(
+    states=GaussianState([10, 0, 80, 0],
+                         np.diag([1, 0, 1, 0])),
+    position_mapping=(0, 2),
+    sensors=[radar1])
+
+# Instantiate the second sensor platform with the second sensor
+sensor2_platform = FixedPlatform(
+    states=GaussianState([75, 0, 10, 0],
+                         np.diag([1, 0, 1, 0])),
+    position_mapping=(0, 2),
+    sensors=[radar2])
+
+# create copy of the simulators, in this way all trackers will receive the same detections
+gt_sims = tee(groundtruth_simulation, 2)
+
+# create the radar simulators
+radar_simulator1 = PlatformDetectionSimulator(
+    groundtruth=gt_sims[0],
+    platforms=[sensor1_platform])
+
+radar_simulator2 = PlatformDetectionSimulator(
+    groundtruth=gt_sims[1],
+    platforms=[sensor2_platform])
+
+radar1plot, radar1KF, radar1PF = tee(radar_simulator1, 3)
+radar2plot, radar2KF, radar2PF = tee(radar_simulator2, 3)
+
+# %%
+# Visualise the detections from the sensors
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# Before preparing the different trackers components, we can visualise the target ground truth and
+# its detections from the two sensors.
+# In this way we can appreciate how the measurements are different and can lead to separate tracks.
+
+from stonesoup.plotter import Plotterly
+from stonesoup.metricgenerator.tracktotruthmetrics import SIAPMetrics
+from stonesoup.measures import Euclidean
+from stonesoup.plotter import MetricPlotter
+from stonesoup.dataassociator.tracktotrack import TrackToTruth
+from stonesoup.metricgenerator.manager import MultiManager
+
+# Lists to hold the detections from each sensor
+s1_detections = []
+s2_detections = []
+
+groundtruth_generator = groundtruth_simulation.groundtruth_paths_gen()
+
+truths = set()
+# Iterate over the time steps, extracting the detections and truths
+for (time, sd1), (_, sd2) in zip(radar1plot, radar2plot):
+    s1_detections.append(sd1)
+    s2_detections.append(sd2)
+    truths.update(next(groundtruth_generator)[1])  # consider only the path
+
+# Plot the detections from the two radars
+plotter = Plotterly()
+plotter.plot_measurements(s1_detections, [0, 2], marker=dict(color='orange', symbol='305'),
+                          measurements_label='Sensor 1 measurements')
+plotter.plot_measurements(s2_detections, [0, 2], marker=dict(color='blue', symbol='0'),
+                          measurements_label='Sensor 2 measurements')
+plotter.plot_sensors({sensor1_platform, sensor2_platform}, [0, 1],
+                     marker=dict(color='black', symbol='1', size=10))
+plotter.plot_ground_truths(truths, [0, 2])
+plotter.fig
```

Comment:
This plot is a little confusing, as some as sensor 1 detections look very similar to clutter symbols we use in other examples, and equally clutter looks like detections we use in other examples. Should raise an issue to make this easier but for now, to keep consisent with other examples, maybe we should do this:

```suggestion
from stonesoup.plotter import Plotterly
from stonesoup.metricgenerator.tracktotruthmetrics import SIAPMetrics
from stonesoup.measures import Euclidean
from stonesoup.plotter import MetricPlotter
from stonesoup.dataassociator.tracktotrack import TrackToTruth
from stonesoup.metricgenerator.manager import MultiManager
from stonesoup.types.detection import Clutter

# Lists to hold the detections from each sensor
s1_detections = []
s2_detections = []

groundtruth_generator = groundtruth_simulation.groundtruth_paths_gen()

truths = set()
# Iterate over the time steps, extracting the detections and truths
for (time, sd1), (_, sd2) in zip(radar1plot, radar2plot):
    s1_detections.append(sd1)
    s2_detections.append(sd2)
    truths.update(next(groundtruth_generator)[1])  # consider only the path

# Plot the detections from the two radars
plotter = Plotterly()
plotter.plot_measurements([d for ds in s1_detections for d in ds if not isinstance(d, Clutter)],
                          [0, 2], marker=dict(color='red'), measurements_label='Sensor 1 measurements')
plotter.plot_measurements([d for ds in s1_detections for d in ds if isinstance(d, Clutter)],
                          [0, 2], marker=dict(color='red', symbol='star-triangle-up'), measurements_label='Sensor 1 measurements')
plotter.plot_measurements([d for ds in s2_detections for d in ds if not isinstance(d, Clutter)],
                          [0, 2], marker=dict(color='blue'), measurements_label='Sensor 2 measurements')
plotter.plot_measurements([d for ds in s2_detections for d in ds if isinstance(d, Clutter)],
                          [0, 2], marker=dict(color='blue', symbol='star-triangle-up'), measurements_label='Sensor 2 measurements')
plotter.plot_sensors({sensor1_platform, sensor2_platform}, [0, 1],
                     marker=dict(color='black', symbol='1', size=10))
plotter.plot_ground_truths(truths, [0, 2])
plotter.fig
```

Annoyingly verbose, but plot is easier to interpret now I think.

Reply:
Yes indeed, I tried some solutions on how to differentiate the clutter and detections from two sensors but I was not obtaining what I wanted. I'll use this solution and raise an issue pointing to this specific case as example of trying to differentiate clutter from different sensors.